test/javascripts/acceptance/category-chooser-initializer-test.js

```diff
@@ -0,0 +1,21 @@
+import { click, visit } from "@ember/test-helpers";
+import { acceptance } from "discourse/tests/helpers/qunit-helpers";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+import { test } from "qunit";
+
+acceptance("CategoryChooser", function (needs) {
+  needs.user();
+  needs.settings({
+    allow_uncategorized_topics: false,
+  });
+
+  test("does not display category with custom_wizard_hide_from_composer set to 't'", async function (assert) {
```

Comment:
@jumagura Is this testing anything now that the setting is removed?

Reply:
That test needed refactoring. Now it reviews the composer and checks if filtering is working correctly